test/linalg/bidiag.jl

```diff
@@ -23,6 +23,8 @@ for relty in (Float32, Float64, BigFloat), elty in (relty, Complex{relty})
     debug && println("Test constructors")
     @test Bidiagonal(dv,ev,'U') == Bidiagonal(dv,ev,true)
     @test_throws ArgumentError Bidiagonal(dv,ev,'R')
+    @test_throws DimensionMismatch Bidiagonal(dv,ones(elty,n),true)
+    @test_throws ErrorException Bidiagonal(dv,ev)
```

Comment:
may as well? it can be a separate commit since it's not like the current state will fail tests